File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        res = [None]*(len(nums)*2)
        for i in range(len(nums)):
            res[i] = nums[i]
            res[i+len(nums)] = nums[i]
        return res

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int res = 0;
        int currMin = prices[0];
        for (int i = 1; i < prices.length; i++) {
            int profit = prices[i] - currMin;
            res = Math.max(res, profit);
            currMin = Math.min(currMin, prices[i]);
        }
        return res;
    }
}
